app/services/pdf_writer.rb

```diff
@@ -8,14 +8,15 @@ class PDFWriter
   FONT_OPTIONS = {size: 7, align: :center}
   TABLE_OPTIONS = {width: 90, align: :center, border_width: 0.5}
 
-  def initialize(user)
+  def initialize(user, years:)
```

Comment:
I actually have no idea what this does. Is the param a symbol?

Reply:
It's a required keyword argument:  https://robots.thoughtbot.com/ruby-2-keyword-arguments#required-keyword-arguments